### DISCUSSION TITLE
Custom domains for emails

### DISCUSSION THREAD

**Discussion by ptcampbell** [supabase/supabase#2228] [Questions] [answered]
Hi! 

Is it possible to use my own domain for Supabase generated emails? If not, is this on the roadmap? Emails from `noreply@mail.app.supabase.io` don't look very trustworthy. 

From a product perspective, it would be cool if you could connect to an external mail service such as Mailgun, Sendgrid etc. from the Supabase UI itself. 

Cheers.

**Comment by vicovaro** [comment]
If I remember well, it is part of supabase paid that you can use custom domain.

**Comment by itechnica** [comment, marked as answer]
Custom mail server requires a paid plan; see https://supabase.io/pricing
For mail server configuration see https://github.com/supabase/gotrue#e-mail

**Comment by ptcampbell** [comment]
Awesome. Thanks!
